portafly/src/i18n/locales/en/accounts.yml

```diff
@@ -27,19 +27,22 @@ team_members_filter:
   by_name: Name
   by_email: Email
   by_sso_authentication: SSO authentication
-   yes: Yes
-   no: Not yet
+  by_sso_authentication_options:
+    yes: Yes
+    no: Not yet
```

Comment:
@porueesq @dfennessy 
In yaml format, where you have indentation it means the indented items are children.
In this case `yes` and `no` are children of `by_sso_authentication`
The parent `by_sso_authentication`, can't have a value when it has children

Reply:
@damianpm 
> In this case yes and no are children of by_sso_authentication
Exactly, by_sso_authentication is a filter criterion that has two possible values: Yes and Not yet

> The parent by_sso_authentication, can't have a value when it has children
Not sure what you mean.

Reply:
@porueesq 
Yes, we need an item for the label (by_sso_authentication), and one for the options (by_sso_authentication_options).
The second one, as it has children, can't have a value, it means that you can't  do this:
```
  by_sso_authentication: SSO authentication
    yes: Yes
    no: Not yet
```
In other words, you can't assign "SSO authentication" to "by_sso_authentication", that's possible only when the item has no children

To give you a better idea, it will look like this
![filters](https://user-images.githubusercontent.com/13486237/82887258-0c3ce300-9f48-11ea-899f-239a9eaa646a.png)

Reply:
@damianpm Got you. Then these lines should be available in the file:

```
by_sso_authentication: SSO authentication
by_sso_authentication_description: Label for the filter criterion based on SSO authentication

by_sso_authentication_options: 
by_sso_authentication_options_description: Group name that contains the option of the SSO authentication criterion
 yes: Yes
 no: Not yet
```